find_me_a_sandwich/app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,22 @@
+class FavoritesController < ApplicationController
+  def show
+    params[:id] ||= current_user.id if user_signed_in?
+
+    @favorites = Favorite.where(user_id: params[:id])
+  end
+
+  def new
```

Comment:
This should be `#create`, not `#new`.  `#new` is for creation forms (typically a GET action), `#create` is the receiver of the POST from that form.